openapi-cli/src/test/resources/expected_gen/petstore_service_swagger.bal

```diff
@@ -0,0 +1,46 @@
+import ballerina/http;
+
+listener http:Listener ep0 = new (443, config = {host: "petstore.swagger.io"});
+
+service /v2 on ep0 {
+    resource function put pet(@http:Payload {mediaType: ["application/json", "application/xml"]} Pet payload) returns http:BadRequest|http:NotFound|http:MethodNotAllowed {
+    }
+    resource function post pet(@http:Payload {mediaType: ["application/json", "application/xml"]} Pet payload) returns http:MethodNotAllowed {
+    }
+    resource function get pet/findByStatus(string[] status) returns Pet[]|http:BadRequest {
```

Comment:
Missing blank lines in-between resource functions in the generated service

Reply:
It is how the service is getting generated from the tool. There is no new line between resource functions.